exploring_UV_extinction_curves_pt1/downloading_UV_data_pt1.ipynb

```diff
@@ -0,0 +1,1433 @@
+{
```

Comment:
<p>Might just be a wording thing, but "a complete image" confused me at first because I was like aren't these spectra, not images? Perhaps "in order to have a more complete understanding of the extinction curve across a broad range of wavelengths,"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ScienceBetter/notebooks-stsci/pull/14/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>